app/views/devices_guidance/index.html.erb

```diff
@@ -13,7 +13,7 @@
     </h1>
 
     <p class="govuk-body">
-      Ordering is now closed for laptops and tablets.
+      Ordering is now closed for laptops and tablets. <span class="app-no-wrap">4G wireless routers</span> are available to order until <span class="app-no-wrap">31 July</span>.
```

Comment:
Change line 16 to: Ordering is closed for laptops, tablets and extra mobile data. 4G wireless routers are available to order until 31 July.